galleries/plot_types/3D/plot3d_simple.py

```diff
@@ -0,0 +1,27 @@
+"""
+================
+plot(xs, ys, zs)
```

Comment:
```suggestion
plot(x, y, z)
```
For consistency. Best also change the variable names below.

oh, I see, the existing scatter() was using this convention, but that's really the odd ball and should be changed as well.

Reply:
The 3D `plot` function has `xs`, `ys`, and `zs` as the argument names, I figure it's best to be consistent with that?
https://matplotlib.org/stable/api/_as_gen/mpl_toolkits.mplot3d.axes3d.Axes3D.plot.html#mpl_toolkits.mplot3d.axes3d.Axes3D.plot

Reply:
Add it to the list of annoying API inconsistencies